api/cpp/include/slint-testing.h

```diff
@@ -205,6 +205,18 @@ class ElementHandle
         }
     }
 
+    /// Returns the value of the element's `accessible-role` property, if present. Use this property
+    /// to locate elements by their type/role, i.e. buttons, checkboxes, etc.
+    std::optional<slint::AccessibleRole> accessible_role() const
```

Comment:
In rust, the accessible role is in the testing namespace. 

Is it just because it's harder to put the enum in the public namespace?

Reply:
Yes, but... I can try to make it work :)